AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -78,7 +86,11 @@ extension MicrophoneManager: AVAudioRecorderDelegate {
     func audioRecorderBeginInterruption(_ recorder: AVAudioRecorder) {
         Log.warning("audio recorder interruption began")
         levelTimer?.invalidate()
-        try! measurementStreamStorage.updateSessionStatus(.DISCONNECTED, for: session!.uuid)
+        measurementStreamStorage.accessStorage { [self] storage in
```

Comment:
strong capturing `[self]` is redundant (seems like `self` is not used inside too)

Reply:
It seems like it but when I remove it I get en error "Reference to property 'session' in closure requires explicit use of 'self' to make capture semantics explicit". That's why it's there :)

Reply:
I can also have it here, not sure if it makes a difference: 
`                try! storage.updateSessionStatus(.DISCONNECTED, for: self.session!.uuid)
`